java/org/apache/tomcat/util/net/SSLUtilBase.java

```diff
@@ -122,6 +122,10 @@ protected SSLUtilBase(SSLHostConfigCertificate certificate, boolean warnTls13) {
         sslHostConfig.setTls13RenegotiationAvailable(isTls13RenegAuthAvailable());
 
         // Calculate the enabled ciphers
+        boolean warnOnSkip = true;
+        if (sslHostConfig.getCiphers().equals(sslHostConfig.DEFAULT_TLS_CIPHERS)) {
+            warnOnSkip = false;
+        }
```

Comment:
`boolean warnOnSkip = !sslHostConfig.getCiphers().equals(sslHostConfig.DEFAULT_TLS_CIPHERS);`

And move this inside the block where it is used.